# S2648

## About

<font size=4>

**S2648.txt** is forward mutations in S5296.

**RS2297.txt** is the subset of S2648 that can be mapped to multiple protein structures.

**MS2297.txt** is the results of S2297 using homology modeled structures.

**Selected Structures.rar** is the crystal structures in S2648.

**Redundant Structures.rar** is the crystal structures in RS2297.

**Modeled Structures.rar** is the modeled structures in MS2297.

</font> 

## Terms in the files

<font size=4>

PDB Id: The PDB entry of the protein.

Mutated Chain: Protein chain with mutation.

Mutation_PDB: The mutation corresponding to the residue numbering found in the protein databank. The first character is one letter amino acid code for the wild-type residue, the second to penultimate characters indicate residue number, and the final character indicates the mutant amino acid.

UniProt: The UniProt ID of the protein.

Mutation_UNP: The mutation corresponding to the residue numbering found in the protein sequence.

DDGexp: Experimental changes of unfolding Gibbs free energy upon mutations (in kcal/mol).

Location: Location of the mutated site on the protein.

PremPS: Predicted changes of unfolding free energy upon mutations by PremPS (in kcal/mol). Positive and negative sign corresponds to destabilizing and stabilizing mutations, respectively. 

PremPS(CV4): Predicted changes of unfolding free energy upon mutations by PremPS performing "leave-one-protein-out" cross-validation (in kcal/mol). Positive and negative sign corresponds to destabilizing and stabilizing mutations, respectively. 

Resolution: Resolution of the crystal structure.

METHOD: Experimental method of the crystal structure.

Aggregate state: The aggregate state of the crystal structure.

weight: Total structure weight of the crystal structure.

Sequence Identity: Sequence identity between target sequences and templates.

RMSD: The root mean square deviation of the coordinates of all CÎ± atoms between crystal and modeled structures.

<font>